wp-content/themes/amplify/style.css

```diff
@@ -0,0 +1,9 @@
+/*
+Theme Name:     Amplify
+Theme URI:      https://github.com/INN/umbrella-inndev/tree/master/wp-content/themes/amplify/
+Description:    Largo child theme.
```

Comment:
@benlk Can we update this to something more descriptive? Other than that, it looks beautiful!

Reply:
Drawing from https://inn.org/project/amplify-news-project/ , perhaps "Amplify News Project" and "Largo child theme for Amplify, with enhanced support for Link Roundups"?

Reply:
> Largo child theme for Amplify, with enhanced support for Link Roundups

That one sounds better, but up to you 😬

Reply:
Updated in 4c9b24b